File: Interview_Prep__App/native_java/src/main/java/com/dreams/androidquizapp/repository/reponseModels/QuestionsResponse.java
package com.dreams.androidquizapp.repository.reponseModels;

import com.dreams.androidquizapp.repository.models.Question;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Tc2r on 5/19/2017.
 * <p>
 * Description:
 */

public class QuestionsResponse {
    @SerializedName("questions")
    private ArrayList<Question> questionsList;

    public ArrayList<Question> getQuestionsList() {
        return questionsList;
    }
}

File: Interview_Prep_App/native_kotlin/retrofit/src/main/java/com/interviewprep/kotlinretrofit/repository/models/response/QuestionsResponse.kt
package com.interviewprep.kotlinretrofit.repository.models.response

import com.google.gson.annotations.SerializedName
import com.interviewprep.kotlinretrofit.repository.models.Question

data class QuestionsResponse(
    @SerializedName("questions")
    var questions: ArrayList<Question>
)
